addon/utils/listener-name.js

```diff
@@ -1,8 +1,14 @@
+import getCmdKey from 'ember-keyboard/utils/get-cmd-key';
+
 function sortedKeys(keyArray) {
   return keyArray.sort().join('+');
 }
 
 export default function listenerName(type, keyArray = []) {
+  if (keyArray.indexOf('cmd') > -1) {
+    keyArray[keyArray.indexOf('cmd')] = getCmdKey();
+  }
```

Comment:
I originally was going to add this in the `handleKeyEvent` logic, but it didn't quite work out like I expected. I didn't want to break other people's listeners for `meta` and other keys, and `handleKeyEvent` doesn't know anything about the listeners. It seemed better to do the rewrite right away and keep the other behavior the same.